src/Tribe/Integrations/WPML/Meta.php

```diff
@@ -38,6 +38,14 @@ public function translate_post_id( $value, $object_id, $meta_key ) {
 			return $value;
 		}
 
+		$cache = tribe_cache();
+
+		$cache_key = $object_id . '-' . $meta_key;
```

Comment:
Salt the cache  key a bit more, it will end up in a pretty used pool and the chances of a name conflict are pretty high:
```suggestion
		$cache_key = 'wpml_meta_translate_post_id_' . $object_id . '-' . $meta_key;
```